src/core_modules/capture-core/components/D2Form/D2Section.component.js

```diff
@@ -15,6 +15,7 @@ const getStyles = theme => ({
     containerCustomForm: {
         paddingTop: 10,
         paddingBottom: 10,
+        overflowX: 'scroll',
```

Comment:
Have you checked if there are any difference between overflow-scroll and overflow-auto here? To my knowledge, `scroll` always puts on a scroll bar, even though the form is not overflowing.

Reply:
Good point, @eirikhaugstulen! I have changed to `overflowX: 'auto'`